src/Entity/Customer.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Entity;
+
+use App\Model\Message;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity
+ */
+class Customer
```

Comment:
Column names are not necessary if there is no particular reason to change column names in MySQL database. We can let Doctrine to take care of this. This way we can avoid(_) everywhere to keep consistent camel case style.
There is no particular reason to keep repositories in entity repository. Why not to use a Repository directory instead?
There is no repository linking annotation to link this entity to CustomerRepository.
Also, primary key column is missing in the entity.